### DISCUSSION TITLE
Links to Korp installations (sites)?

### DISCUSSION THREAD

**Discussion by janiemi** [spraakbanken/korp-frontend#298] [General]
I think it would be nice if we had somewhere a list of links to different Korp installations (sites), as at least I haven’t found one. What would be a good place for such a list? The [`README.md`](https://github.com/spraakbanken/korp-frontend/blob/dev/README.md) file or another documentation file linked from `README.md`? Or a Wiki page, if GitHub Wiki were enabled?

Here is a list of Korp installations I know of and that I now found:

- [Språkbanken Text](https://spraakbanken.gu.se/korp/)
- [The Language Bank of Finland (Kielipankki)](https://korp.csc.fi) (the URL will change in the near future)
- [Iceland / Stofnun Árna Magnússonar í íslenskum fræðum](https://malheildir.arnastofnun.is/)
- [Tromsø / Giellatekno](https://gtweb.uit.no/korp/)
- [Copenhagen / Institut for Nordiske Studier og Sprogvidenskab](https://alf.hum.ku.dk/korp/)

In addition, Korp has been at least tested in Estonia and at the University of Alberta ALTLab, which have forked  Korp in GitHub, but I couldn’t find the actual Korp sites now. (Which is one reason why I’d find it nice to have such a list, even though it might eventually get out of date.)

And what about suggesting adding a link to Korp on the [CWB page for demos under “Public Web interfaces using CWB”](https://cwb.sourceforge.io/demos.php#public). The page currently has a link to a [CWB interface for PAROLE and SUC at Språkbanken](https://spraakbanken.gu.se/parole/) that appears to exist but is no longer maintained, but not to any Korp installation. I think the page could perhaps link to Språkbanken’s Korp or to a list of different Korp installations or both, or perhaps to GitHub. The CWB web site is not very actively maintained, though, so it might take some time before a link would be added there.

**Comment by majsan** [comment]
A first step could be adding links to the instances in README.md in this repo. But what about actually adding them to the frontend somewhere? Maybe it would be bad because URL:s change and old instances of Korp will show the wrong URL.

I will bring it up internally and see if we can get a redirect for the URL on the CWB homepage, since it is broken anyway. But I can also contact someone about actually changing the link.

**Comment by majsan** [comment]
I added the links to the frontend README. When your URL changes, you can let me know or send a pull request.

**Reply by janiemi** [reply]
Looks good. We’ll inform you when our URL changes. (The old one will be redirected, but I think it’s nevertheless better to link to the up-to-date URL.)

**Comment by majsan** [comment]
I think CWB removed the demo page from the site...
